tests/integration/server/api/users_api_test.py

```diff
@@ -426,3 +426,47 @@ def test_reject_users_proposals():
 
         assert admin
         assert not non_admin
+
+
+def test_update_user_password():
```

Comment:
should this be two different tests?

Reply:
By doing it as one continuous you can show that the changes took place because we are able to login with the newly changed password.  If we don't then we don't have the same logic flow and will make the test unstable.  One way to get around this would be to access the database directly to verify, but I believe it is a better test to actually utilized the login capability as if it is one continuous path in a single test.  This will also prevent it from being unstable via rapid changes.

Reply:
oh no, I agree with that, I meant lines 455-472, where you try to change a user's password as a non-administrator and assert a 400 response.

Reply:
Based on conversation the value of splitting is not worth the effort based on the Integration test Epic we will be hitting soon.